v3/tutorials/02-proof-of-existence/index.mdx

```diff
@@ -199,10 +199,10 @@ To implement the pallet events:
    #[pallet::event]
    #[pallet::generate_deposit(pub(super) fn deposit_event)]
    pub enum Event<T: Config> {
-     /// Event emitted when a proof has been claimed. [who, claim]
-     ClaimCreated(T::AccountId, Vec<u8>),
-     /// Event emitted when a claim is revoked by the owner. [who, claim]
-     ClaimRevoked(T::AccountId, Vec<u8>),
+       /// Event emitted when a proof has been claimed. [who, claim]
```

Comment:
For rust snippets we want to use `hard tabs`, not spaces :pray:

Reply:
Hmm... I guess that leaves you with two choices: You can indent enumerations with 3 spaces, but then the rust code blocks will mix tabs and spaces. Or you can indent enumerations with 1 tab, which prettier will replace with three spaces everytime you run it, forcing you to put a `<!-- prettier-ignore -->` comment before every enumeration. Since half of each tutorial is contained in an enumeration, I'd go for the first option.

Reply:
Okay, this is resolved in 4c78cb1.

You should note, however, that markdown renders the tabs in the Rust code blocks as 4 spaces. In particular, when copying the code using the _Copy_ button, you'll get spaces instead of tabs. Someone copying the code into files using the Substrate code style will get mixed spaces/tabs.

You should also run prettier on some of these changes. I didn't do that in order to not complicate the merge any further.

Another thing is that many code blocks (now?) seem to require a horizontal scroll bar. I'd try to fix that by extending the geometry of the document (for code blocks _only_).